setup.cfg

```diff
@@ -20,3 +20,5 @@ install_requires =
     gym
     wheel
     pygame
+    pytest
+    pytest-mock
```

Comment:
I think we should create a separate `requirement.txt` for development-purposed dependencies because this list here should contain dependencies that are required when the users running our package. [This](https://stackoverflow.com/questions/43658870/requirements-txt-vs-setup-py#:~:text=The%20short%20answer%20is%20that,you%20would%20only%20need%20requirements.) is also a good reference I read the other day.

Reply:
Done.